spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_structural_common.py

```diff
@@ -418,7 +419,53 @@ def __write_post_to_pre_table(
             numpy.concatenate(padded_rows).T, formats="u1, u1, u2").view("u4")
         spec.write_array(post_to_pre)
 
-    def get_parameters_sdram_usage_in_bytes(
+    def get_actual_parameters_sdram_usage_in_bytes(
```

Comment:
As far as I can tell, this is the same as get_parameters_sdram_usage_in_bytes(). If this is called after partitioning is complete I would hope this would indeed return the actual SDRAM required. Have you run any tests for this?

Reply:
This was tested with https://github.com/SpiNNakerManchester/sPyNNaker8/pull/440
However I don't think any of these include the case of two  structurally plastic connections (each from a different source) 
into the same vertex
@pabogdan do you have an example script that has such a script.
Also if you have other scripts that you are keen to include in the test please add then either in the above spy8 pr or in a separate pr.